README.md

```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] It will be great to update your readme file with the new updated readme template provided to you by Microverse. This will ensure that your project is more professional. This is the link to the newly updated[ readme](https://github.com/microverseinc/readme-template)

Reply:
Thank you. I have already modified the readme according to Microverse rules.